src/auth/models.cpp

```diff
@@ -370,6 +399,14 @@ Permissions &Role::permissions() { return permissions_; }
 const FineGrainedAccessHandler &Role::fine_grained_access_handler() const { return fine_grained_access_handler_; }
 FineGrainedAccessHandler &Role::fine_grained_access_handler() { return fine_grained_access_handler_; }
 
+const FineGrainedAccessPermissions &Role::GetFineGrainedAccessLabelPermissions() const {
```

Comment:
Do we need GetFineGrainedAccessLabelPermissions or GetFineGrainedAccessEdgeTypePermissions?